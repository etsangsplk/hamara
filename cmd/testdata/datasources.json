[
  {
    "access": "",
    "basicAuth": false,
    "database": "",
    "id": 25,
    "isDefault": false,
    "jsonData": {
      "authType": "credentials",
      "defaultRegion": "eu-west-2"
    },
    "name": "gdev-cloudwatch",
    "orgId": 1,
    "password": "",
    "readOnly": false,
    "type": "cloudwatch",
    "typeLogoUrl": "public/app/plugins/datasource/cloudwatch/img/amazon-web-services.png",
    "url": "",
    "user": ""
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "[logs-]YYYY.MM.DD",
    "id": 9,
    "isDefault": false,
    "jsonData": {
      "esVersion": 2,
      "interval": "Daily",
      "timeField": "@timestamp"
    },
    "name": "gdev-elasticsearch-v2-logs",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "elasticsearch",
    "typeLogoUrl": "public/app/plugins/datasource/elasticsearch/img/elasticsearch.svg",
    "url": "http://localhost:9200",
    "user": ""
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "[metrics-]YYYY.MM.DD",
    "id": 8,
    "isDefault": false,
    "jsonData": {
      "esVersion": 2,
      "interval": "Daily",
      "timeField": "@timestamp"
    },
    "name": "gdev-elasticsearch-v2-metrics",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "elasticsearch",
    "typeLogoUrl": "public/app/plugins/datasource/elasticsearch/img/elasticsearch.svg",
    "url": "http://localhost:9200",
    "user": ""
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "[logs-]YYYY.MM.DD",
    "id": 11,
    "isDefault": false,
    "jsonData": {
      "esVersion": 5,
      "interval": "Daily",
      "timeField": "@timestamp"
    },
    "name": "gdev-elasticsearch-v5-logs",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "elasticsearch",
    "typeLogoUrl": "public/app/plugins/datasource/elasticsearch/img/elasticsearch.svg",
    "url": "http://localhost:10200",
    "user": ""
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "[metrics-]YYYY.MM.DD",
    "id": 10,
    "isDefault": false,
    "jsonData": {
      "esVersion": 5,
      "interval": "Daily",
      "timeField": "@timestamp"
    },
    "name": "gdev-elasticsearch-v5-metrics",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "elasticsearch",
    "typeLogoUrl": "public/app/plugins/datasource/elasticsearch/img/elasticsearch.svg",
    "url": "http://localhost:10200",
    "user": ""
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "[filebeat-]YYYY.MM.DD",
    "id": 14,
    "isDefault": false,
    "jsonData": {
      "esVersion": 60,
      "interval": "Daily",
      "timeField": "@timestamp"
    },
    "name": "gdev-elasticsearch-v6-filebeat",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "elasticsearch",
    "typeLogoUrl": "public/app/plugins/datasource/elasticsearch/img/elasticsearch.svg",
    "url": "http://localhost:11200",
    "user": ""
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "[logs-]YYYY.MM.DD",
    "id": 13,
    "isDefault": false,
    "jsonData": {
      "esVersion": 60,
      "interval": "Daily",
      "timeField": "@timestamp"
    },
    "name": "gdev-elasticsearch-v6-logs",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "elasticsearch",
    "typeLogoUrl": "public/app/plugins/datasource/elasticsearch/img/elasticsearch.svg",
    "url": "http://localhost:11200",
    "user": ""
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "[metrics-]YYYY.MM.DD",
    "id": 12,
    "isDefault": false,
    "jsonData": {
      "esVersion": 60,
      "interval": "Daily",
      "timeField": "@timestamp"
    },
    "name": "gdev-elasticsearch-v6-metrics",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "elasticsearch",
    "typeLogoUrl": "public/app/plugins/datasource/elasticsearch/img/elasticsearch.svg",
    "url": "http://localhost:11200",
    "user": ""
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "[filebeat-]YYYY.MM.DD",
    "id": 17,
    "isDefault": false,
    "jsonData": {
      "esVersion": 70,
      "interval": "Daily",
      "logLevelField": "fields.level",
      "logMessageField": "message",
      "timeField": "@timestamp",
      "timeInterval": "10s"
    },
    "name": "gdev-elasticsearch-v7-filebeat",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "elasticsearch",
    "typeLogoUrl": "public/app/plugins/datasource/elasticsearch/img/elasticsearch.svg",
    "url": "http://localhost:12200",
    "user": ""
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "[logs-]YYYY.MM.DD",
    "id": 16,
    "isDefault": false,
    "jsonData": {
      "esVersion": 70,
      "interval": "Daily",
      "timeField": "@timestamp"
    },
    "name": "gdev-elasticsearch-v7-logs",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "elasticsearch",
    "typeLogoUrl": "public/app/plugins/datasource/elasticsearch/img/elasticsearch.svg",
    "url": "http://localhost:12200",
    "user": ""
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "[metricbeat-]YYYY.MM.DD",
    "id": 18,
    "isDefault": false,
    "jsonData": {
      "esVersion": 70,
      "interval": "Daily",
      "timeField": "@timestamp",
      "timeInterval": "10s"
    },
    "name": "gdev-elasticsearch-v7-metricbeat",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "elasticsearch",
    "typeLogoUrl": "public/app/plugins/datasource/elasticsearch/img/elasticsearch.svg",
    "url": "http://localhost:12200",
    "user": ""
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "[metrics-]YYYY.MM.DD",
    "id": 15,
    "isDefault": false,
    "jsonData": {
      "esVersion": 70,
      "interval": "Daily",
      "timeField": "@timestamp",
      "timeInterval": "10s"
    },
    "name": "gdev-elasticsearch-v7-metrics",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "elasticsearch",
    "typeLogoUrl": "public/app/plugins/datasource/elasticsearch/img/elasticsearch.svg",
    "url": "http://localhost:12200",
    "user": ""
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "",
    "id": 1,
    "isDefault": false,
    "jsonData": {
      "graphiteVersion": "1.1"
    },
    "name": "gdev-graphite",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "graphite",
    "typeLogoUrl": "public/app/plugins/datasource/graphite/img/graphite_logo.png",
    "url": "http://localhost:8180",
    "user": ""
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "site",
    "id": 5,
    "isDefault": false,
    "jsonData": {
      "timeInterval": "15s"
    },
    "name": "gdev-influxdb",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "influxdb",
    "typeLogoUrl": "public/app/plugins/datasource/influxdb/img/influxdb_logo.svg",
    "url": "http://localhost:8086",
    "user": "grafana"
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "telegraf",
    "id": 6,
    "isDefault": false,
    "jsonData": {
      "timeInterval": "10s"
    },
    "name": "gdev-influxdb-telegraf",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "influxdb",
    "typeLogoUrl": "public/app/plugins/datasource/influxdb/img/influxdb_logo.svg",
    "url": "http://localhost:8086",
    "user": "grafana"
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "",
    "id": 26,
    "isDefault": false,
    "jsonData": {},
    "name": "gdev-loki",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "loki",
    "typeLogoUrl": "public/app/plugins/datasource/loki/img/loki_icon.svg",
    "url": "http://localhost:3100",
    "user": ""
  },
  {
    "access": "",
    "basicAuth": false,
    "database": "grafana",
    "id": 21,
    "isDefault": false,
    "jsonData": {},
    "name": "gdev-mssql",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "mssql",
    "typeLogoUrl": "public/app/plugins/datasource/mssql/img/sql_server_logo.svg",
    "url": "localhost:1433",
    "user": "grafana"
  },
  {
    "access": "",
    "basicAuth": false,
    "database": "grafanatest",
    "id": 22,
    "isDefault": false,
    "jsonData": {},
    "name": "gdev-mssql-ds-tests",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "mssql",
    "typeLogoUrl": "public/app/plugins/datasource/mssql/img/sql_server_logo.svg",
    "url": "localhost:1433",
    "user": "grafana"
  },
  {
    "access": "",
    "basicAuth": false,
    "database": "grafana",
    "id": 19,
    "isDefault": false,
    "jsonData": {},
    "name": "gdev-mysql",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "mysql",
    "typeLogoUrl": "public/app/plugins/datasource/mysql/img/mysql_logo.svg",
    "url": "localhost:3306",
    "user": "grafana"
  },
  {
    "access": "",
    "basicAuth": false,
    "database": "grafana_ds_tests",
    "id": 20,
    "isDefault": false,
    "jsonData": {},
    "name": "gdev-mysql-ds-tests",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "mysql",
    "typeLogoUrl": "public/app/plugins/datasource/mysql/img/mysql_logo.svg",
    "url": "localhost:3306",
    "user": "grafana"
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "",
    "id": 7,
    "isDefault": false,
    "jsonData": {
      "tsdbResolution": 1,
      "tsdbVersion": 1
    },
    "name": "gdev-opentsdb",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "opentsdb",
    "typeLogoUrl": "public/app/plugins/datasource/opentsdb/img/opentsdb_logo.png",
    "url": "http://localhost:4242",
    "user": ""
  },
  {
    "access": "",
    "basicAuth": false,
    "database": "grafana",
    "id": 23,
    "isDefault": false,
    "jsonData": {
      "sslmode": "disable"
    },
    "name": "gdev-postgres",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "postgres",
    "typeLogoUrl": "public/app/plugins/datasource/postgres/img/postgresql_logo.svg",
    "url": "localhost:5432",
    "user": "grafana"
  },
  {
    "access": "",
    "basicAuth": false,
    "database": "grafanadstest",
    "id": 24,
    "isDefault": false,
    "jsonData": {
      "sslmode": "disable"
    },
    "name": "gdev-postgres-ds-tests",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "postgres",
    "typeLogoUrl": "public/app/plugins/datasource/postgres/img/postgresql_logo.svg",
    "url": "localhost:5432",
    "user": "grafanatest"
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "",
    "id": 2,
    "isDefault": false,
    "jsonData": {},
    "name": "gdev-prometheus",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "prometheus",
    "typeLogoUrl": "public/app/plugins/datasource/prometheus/img/prometheus_logo.svg",
    "url": "http://localhost:9090",
    "user": ""
  },
  {
    "access": "proxy",
    "basicAuth": false,
    "database": "",
    "id": 3,
    "isDefault": false,
    "jsonData": {},
    "name": "gdev-slow-prometheus",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "prometheus",
    "typeLogoUrl": "public/app/plugins/datasource/prometheus/img/prometheus_logo.svg",
    "url": "http://localhost:3011",
    "user": ""
  },
  {
    "access": "",
    "basicAuth": false,
    "database": "",
    "id": 4,
    "isDefault": true,
    "jsonData": {},
    "name": "gdev-testdata",
    "orgId": 1,
    "password": "",
    "readOnly": true,
    "type": "testdata",
    "typeLogoUrl": "public/app/plugins/datasource/testdata/img/testdata.svg",
    "url": "",
    "user": ""
  }
]
