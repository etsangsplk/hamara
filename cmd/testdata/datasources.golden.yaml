configversion:
  apiVersion: 0
datasources:
- orgId: 1
  version: 0
  name: gdev-cloudwatch
  type: cloudwatch
  access: ""
  url: ""
  password: ""
  user: ""
  database: ""
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    authType: credentials
    defaultRegion: eu-west-2
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-elasticsearch-v2-logs
  type: elasticsearch
  access: proxy
  url: http://localhost:9200
  password: ""
  user: ""
  database: '[logs-]YYYY.MM.DD'
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    esVersion: 2
    interval: Daily
    timeField: '@timestamp'
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-elasticsearch-v2-metrics
  type: elasticsearch
  access: proxy
  url: http://localhost:9200
  password: ""
  user: ""
  database: '[metrics-]YYYY.MM.DD'
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    esVersion: 2
    interval: Daily
    timeField: '@timestamp'
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-elasticsearch-v5-logs
  type: elasticsearch
  access: proxy
  url: http://localhost:10200
  password: ""
  user: ""
  database: '[logs-]YYYY.MM.DD'
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    esVersion: 5
    interval: Daily
    timeField: '@timestamp'
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-elasticsearch-v5-metrics
  type: elasticsearch
  access: proxy
  url: http://localhost:10200
  password: ""
  user: ""
  database: '[metrics-]YYYY.MM.DD'
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    esVersion: 5
    interval: Daily
    timeField: '@timestamp'
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-elasticsearch-v6-filebeat
  type: elasticsearch
  access: proxy
  url: http://localhost:11200
  password: ""
  user: ""
  database: '[filebeat-]YYYY.MM.DD'
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    esVersion: 60
    interval: Daily
    timeField: '@timestamp'
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-elasticsearch-v6-logs
  type: elasticsearch
  access: proxy
  url: http://localhost:11200
  password: ""
  user: ""
  database: '[logs-]YYYY.MM.DD'
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    esVersion: 60
    interval: Daily
    timeField: '@timestamp'
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-elasticsearch-v6-metrics
  type: elasticsearch
  access: proxy
  url: http://localhost:11200
  password: ""
  user: ""
  database: '[metrics-]YYYY.MM.DD'
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    esVersion: 60
    interval: Daily
    timeField: '@timestamp'
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-elasticsearch-v7-filebeat
  type: elasticsearch
  access: proxy
  url: http://localhost:12200
  password: ""
  user: ""
  database: '[filebeat-]YYYY.MM.DD'
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    esVersion: 70
    interval: Daily
    logLevelField: fields.level
    logMessageField: message
    timeField: '@timestamp'
    timeInterval: 10s
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-elasticsearch-v7-logs
  type: elasticsearch
  access: proxy
  url: http://localhost:12200
  password: ""
  user: ""
  database: '[logs-]YYYY.MM.DD'
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    esVersion: 70
    interval: Daily
    timeField: '@timestamp'
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-elasticsearch-v7-metricbeat
  type: elasticsearch
  access: proxy
  url: http://localhost:12200
  password: ""
  user: ""
  database: '[metricbeat-]YYYY.MM.DD'
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    esVersion: 70
    interval: Daily
    timeField: '@timestamp'
    timeInterval: 10s
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-elasticsearch-v7-metrics
  type: elasticsearch
  access: proxy
  url: http://localhost:12200
  password: ""
  user: ""
  database: '[metrics-]YYYY.MM.DD'
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    esVersion: 70
    interval: Daily
    timeField: '@timestamp'
    timeInterval: 10s
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-graphite
  type: graphite
  access: proxy
  url: http://localhost:8180
  password: ""
  user: ""
  database: ""
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    graphiteVersion: "1.1"
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-influxdb
  type: influxdb
  access: proxy
  url: http://localhost:8086
  password: ""
  user: grafana
  database: site
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    timeInterval: 15s
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-influxdb-telegraf
  type: influxdb
  access: proxy
  url: http://localhost:8086
  password: ""
  user: grafana
  database: telegraf
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    timeInterval: 10s
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-loki
  type: loki
  access: proxy
  url: http://localhost:3100
  password: ""
  user: ""
  database: ""
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData: {}
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-mssql
  type: mssql
  access: ""
  url: localhost:1433
  password: ""
  user: grafana
  database: grafana
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData: {}
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-mssql-ds-tests
  type: mssql
  access: ""
  url: localhost:1433
  password: ""
  user: grafana
  database: grafanatest
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData: {}
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-mysql
  type: mysql
  access: ""
  url: localhost:3306
  password: ""
  user: grafana
  database: grafana
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData: {}
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-mysql-ds-tests
  type: mysql
  access: ""
  url: localhost:3306
  password: ""
  user: grafana
  database: grafana_ds_tests
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData: {}
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-opentsdb
  type: opentsdb
  access: proxy
  url: http://localhost:4242
  password: ""
  user: ""
  database: ""
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    tsdbResolution: 1
    tsdbVersion: 1
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-postgres
  type: postgres
  access: ""
  url: localhost:5432
  password: ""
  user: grafana
  database: grafana
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    sslmode: disable
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-postgres-ds-tests
  type: postgres
  access: ""
  url: localhost:5432
  password: ""
  user: grafanatest
  database: grafanadstest
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData:
    sslmode: disable
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-prometheus
  type: prometheus
  access: proxy
  url: http://localhost:9090
  password: ""
  user: ""
  database: ""
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData: {}
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-slow-prometheus
  type: prometheus
  access: proxy
  url: http://localhost:3011
  password: ""
  user: ""
  database: ""
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: false
  jsonData: {}
  secureJsonData: {}
  editable: false
- orgId: 1
  version: 0
  name: gdev-testdata
  type: testdata
  access: ""
  url: ""
  password: ""
  user: ""
  database: ""
  basicAuth: false
  basicAuthUser: ""
  basicAuthPassword: ""
  withCredentials: false
  isDefault: true
  jsonData: {}
  secureJsonData: {}
  editable: false
deleteDatasources: []

